### PR TITLE
Removed newton3 parameter from all iteratePairwise methods in the containers

### PR DESCRIPTION
**Pull request by HobbyProgrammer**
# Description
Removed newton 3 parameter from all iteratePairwise methods in the containers.

## Related Pull Requests

None

## Resolved Issues

None

# How Has This Been Tested?
Jenkins

### CODE REVIEW

File: src/autopas/containers/directSum/DirectSum.h [hunk 1]
```diff
@@ -78,7 +78,7 @@ class DirectSum : public ParticleContainer<Particle, ParticleCell> {
    * @param useNewton3 whether newton 3 optimization should be used
```
Comment (FG-TUM) [inline src/autopas/containers/directSum/DirectSum.h]: This `@param` needs to be removed

File: src/autopas/containers/verletClusterLists/VerletClusterLists.h [hunk 2]
```diff
@@ -73,7 +73,8 @@ class VerletClusterLists : public ParticleContainer<Particle, FullParticleCell<P
    * @param useNewton3 whether newton 3 optimization should be used
```
Comment (FG-TUM) [inline src/autopas/containers/verletClusterLists/VerletClusterLists.h]: This @param needs to be removed

File: src/autopas/containers/verletListsCellBased/verletListsCells/VerletListsCells.h [hunk 3]
```diff
@@ -72,7 +72,8 @@ class VerletListsCells
    * @param useNewton3 whether newton 3 optimization should be used
```
Comment (FG-TUM) [inline src/autopas/containers/verletListsCellBased/verletListsCells/VerletListsCells.h]: This @param needs to be removed

File: src/autopas/selectors/AutoTuner.h [hunk 4]
```diff
@@ -323,8 +323,7 @@ void AutoTuner<Particle, ParticleCell>::iteratePairwiseTemplateHelper(PairwiseFu
     // @todo remove useNewton3 in iteratePairwise by introducing traversals for DS and VL
```
Comment (FG-TUM) [inline src/autopas/selectors/AutoTuner.h]: This todo is resolved by this PR

**Comment by HobbyProgrammer** [comment]
Thanks, I cleaned it up!

**Pull request merged**